src/PrestaShopBundle/EventListener/ModuleActivatedListener.php

```diff
@@ -140,6 +141,9 @@ public function onKernelController(FilterControllerEvent $event)
      */
     private function showNotificationMessage(ModuleActivated $moduleActivated)
     {
+        if (!($this->session instanceof Session)) {
```

Comment:
already a SessionInterface and it's mandatory. If something is failing, the interface needs to be updated :thinking:

Reply:
But the method `getFlashBag` is only declared in `Session` class and not in the interface.

Reply:
So, we must change the signature of the method if we use undefined methods :thinking:

Reply:
Ping @matks 

I changed the signature of __construct.
I accept a `Session` object and not anymore a `SessionInterface`.
I documented the BC Break.

Ok with that ?

Reply:
I approve this change ✅ . It's a "BC break introduced because the initial code was flawed" so it's all right